### PR TITLE
fix(geo, common): Integrate ngx color

### PR DESCRIPTION
**Pull request by aziz-access**
**What is the current behavior?** (You can also link to an open issue here)

this behavior integrate ngx-color to solve the issues of color alpha

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

**Comment by alecarn** [comment]
@pelord ready for your review

File: packages/geo/src/lib/style/style-modal/layer/style-modal-layer.component.ts [hunk 1]
```diff
@@ -50,12 +48,17 @@ export class StyleModalLayerComponent implements OnInit {
       }
     }
     this.buildStyleData();
+    this.buildForm();
+
+    this.form.valueChanges.subscribe((res) => {
+      console.log(res);
```
Comment (pelord) [inline packages/geo/src/lib/style/style-modal/layer/style-modal-layer.component.ts:54]: Remove this please.

File: packages/geo/src/lib/style/style-modal/drawing/style-modal-drawing.component.ts [hunk 2]
```diff
@@ -39,12 +37,17 @@ export class StyleModalDrawingComponent implements OnInit {
       }
     }
     this.buildStyleData();
+    this.buildForm();
+
+    this.form.valueChanges.subscribe((res) => {
+      console.log(res);
```
Comment (pelord) [inline packages/geo/src/lib/style/style-modal/drawing/style-modal-drawing.component.ts:43]: remove this pls

**Review by pelord — CHANGES_REQUESTED** [review]
Remove console.log pls.

**Pull request merged**